repos.d/gentoo/funtoo.yaml

```diff
@@ -2,45 +2,51 @@
 # Funtoo
 ###########################################################################
 
-# Funtoo - 1.3 Release (Jan 7 2019)
-- name: funtoo_1.3
+# Funtoo - 1.4 Release
+- name: funtoo_1.4
   type: repository
-  desc: Funtoo 1.3
+  desc: Funtoo 1.4
   family: gentoo
   ruleset: [gentoo, funtoo]
   color: '4e4e71'
   minpackages: 35000
   default_maintainer: fallback-mnt-funtoo@repology
   sources:
     - name:
-        - core-gl-kit
-        - core-hw-kit
         - core-kit
-        - core-server-kit
+        - core-hw-kit
         - core-ui-kit
+        - core-server-kit
+        - core-gl-kit
+        - security-kit
+        - xorg-kit
+        - gnome-kit
+        - rust-kit
+        - xfce-kit
+        - kde-kit
         - desktop-kit
-        - dev-kit
+        - media-kit
         - editors-kit
+        - net-kit
+        - text-kit
+        - science-kit
         - games-kit
-        - haskell-kit
+        - perl-kit
         - java-kit
-        - kde-kit
-        - lang-kit
+        - ruby-kit
```

Comment:
You must not duplicate repositories which we handle separately (due to different branch names) below here.

Reply:
should be fixed now